src/SharedMoudule/Components/SideBar/SideBar.tsx

```diff
@@ -84,6 +90,17 @@ export default function SideBar() {
               
               Logout
             </MenuItem>
+
+             {<MenuItem
```

Comment:
We want to adjust this icon and give it the same size as other icons, because it is currently taken of the sidebar.
![image](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/f16feb22-4e4d-43f5-ba22-1aca100eb4c1)